merino/providers/amo/backends/dynamic.py

```diff
@@ -11,6 +11,8 @@
 from merino.providers.amo.backends.protocol import Addon, AmoBackendError
 from merino.utils.http_client import create_http_client
 
+AMO_CONNECT_TIMEOUT = 5.0
```

Comment:
nit: let's perhaps add the `float` type annotation?

```suggestion
AMO_CONNECT_TIMEOUT: float = 5.0
```